repl/repl.go

```diff
@@ -72,7 +73,7 @@ func Interactive(in io.Reader, out io.Writer, options Options) {
 			return
 		}
 		l := prev + scanner.Text()
-		contNeeded := EvalOne(s, macroState, l, out, options)
+		contNeeded, _ := EvalOne(s, macroState, l, out, options)
```

Comment:
Why ignoring errors here?

Maybe it's OK, but a comment would help

Reply:
it's because they've been printed interactively, which is kinda why the spagheti of ifs and modes in EvalOne isn't great, I probably should just copy paste exactly for 3 modes, or ... clean up somehow

agreed on comment, adding

Reply:
https://github.com/grol-io/grol/pull/37/commits/c09493cc02be18a857bf76c8ed1d51ada6561993